src/components/ToggleButtons/ToggleButtons.jsx

```diff
@@ -0,0 +1,44 @@
+import { Component } from 'react';
+import Button from '../Button/Button';
+import { FlexContainer } from '../FlexContainer/FlexContainer';
+import './ToggleButtons.css';
+
+class ToggleButtons extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedButton: 0
+        };
+    }
+
+    handleClick = (index) => {
+        this.setState({ selectedButton: index });
+    };
+
+    render() {
+        const { selectedButton } = this.state;
+        const buttonData = [
```

Comment:
You need to get the product categories from the list of products as an unique values